### DISCUSSION TITLE
Show kernels for variables

### DISCUSSION THREAD

**Discussion by rtaylo45** [idaholab/moose#20536] [Q&A General] [answered]
Hello,

Is there a debug flag, similar to `show_material_props`, which shows which kernels are attached to which variables?

Thanks,
Zack

**Comment by GiudGiud** [comment, marked as answer]
Hello

Dont think so. But verbose output for the kernel_coverage_check would do the trick if you're willing to dive in
https://mooseframework.inl.gov/framework_development/sanity_checking.html#!

Here's the list of all the Debug flags btw
https://mooseframework.inl.gov/syntax/Debug/index.html

Guillaume
